pkg/ruler/api.go

```diff
@@ -0,0 +1,260 @@
+package ruler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/gorilla/mux"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/weaveworks/common/user"
+
+	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/cortexproject/cortex/pkg/util"
+)
+
+// RegisterRoutes registers the configs API HTTP routes with the provided Router.
+func (r *Ruler) RegisterRoutes(router *mux.Router) {
+	for _, route := range []struct {
+		name, method, path string
+		handler            http.HandlerFunc
+	}{
+		{"get_rules", "GET", "/api/prom/api/v1/rules", r.rules},
```

Comment:
In `modules.go` we do `t.server.HTTP.PathPrefix(cfg.HTTPPrefix).Subrouter()` so now it's registered to `/api/prom/api/prom/api/v1/rules`. Here we should remove the `/api/prom` prefix.

Reply:
Good catch, makes it obvious I haven't tested this yet